cmd/containerd-shim-runhcs-v1/pod.go

```diff
@@ -238,10 +239,20 @@ func createPod(ctx context.Context, events publisher, req *task.CreateTaskReques
 		}
 	} else {
 		if isWCOW {
-			// The pause container activation will immediately exit on Windows
+			defaultArgs := "c:\\windows\\system32\\cmd.exe"
+			// For the default pause image, the  entrypoint
+			// used is pause.exe
+			// If the default pause image is not used for pause containers,
+			// the activation will immediately exit on Windows
 			// because there is no command. We forcibly update the command here
-			// to keep it alive.
-			s.Process.CommandLine = "cmd /c ping -t 127.0.0.1 > nul"
+			// to keep it alive only for non-default pause images.
+			// TODO: This override can be completely removed from containerd/1.7
+			if (len(s.Process.Args) == 1 && strings.EqualFold(s.Process.Args[0], defaultArgs)) ||
+				strings.EqualFold(s.Process.CommandLine, defaultArgs) {
+				log.G(ctx).Warning("Detected CMD override for pause container entrypoint." +
+					"Please consider switching to a pause image which an explicit cmd set")
```

Comment:
nit: Should we just say `Please consider switching to a pause image.`? I am not sure what `which an explicit cmd set` means here.

Reply:
+1 I'd log a warning saying something like `Please ensure you are using an image running pause.exe` or something similar